src/apply/value/setValue.js

```diff
@@ -0,0 +1,17 @@
+const Y = require('yjs');
+
+const setValue = (doc, op)=>{
+    if(!doc.hasOwnProperty('data')){
```

Comment:
Looking at the Yjs API docs -- https://github.com/yjs/yjs#shared-types -- it looks to me like you probably want to use .has() here instead of .hasOwnProperty()?

(Though as Chris can attest, my JS-foo is weak, so it's entirely possible that I'm wrong on that.)

Reply:
I found that this check wasn`t really needed because the code overwrites the entire data field, so using the set('data', new Y.Map()) gives the desired behavior.